CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 1.7.1 (2019-08-05)
+
+- Bugfix for correctly deploying major version of static assets. [#51](https://github.com/blackbaud/skyux-deploy/pull/51)
```

Comment:
Fixed the deploy command to correctly deploy the major version of static assets.

Is that suggestion accurate, and is it weird because I said deploy twice?

Reply:
That is accurate and would probably be made less weird if the first instance was surrounded by backticks.